scripts/st2bootstrap-deb.sh

```diff
@@ -460,7 +495,17 @@ install_st2mistral() {
     sudo apt-get install -y st2mistral${ST2MISTRAL_PKG_VERSION}
   else
     sudo apt-get install -y jq
-    PACKAGE_URL="$(curl -Ss -q https://circleci.com/api/v1.1/project/github/StackStorm/${DEV_BUILD}/artifacts | jq -r '.[].url' | egrep "${SUBTYPE}/st2mistral_.*.deb")"
+
+    # Note: We disable global error handler because we want to print a more user-friendly error message
+    set +e
+    PACKAGE_URL=$(get_package_url "${DEV_BUILD}" "${SUBTYPE}" "st2mistral_.*.deb")
+
+    if [ -z "${PACKAGE_URL}" ]; then
+        exit 2
+    fi
+    # Re-enable a global error handler
+    set -e
```

Comment:
Definitely need to reconsider `set +e`/`set -e` here and think about cleaner approach.

Maybe `return` from a function (as noted in previous comment) would help to get rid of this disable/enable `e` hack.